tests/Resources/ti.ui.navigationwindow.addontest.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2017-Present by Axway. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+
+describe.windowsMissing('Titanium.UI.NavigationWindow', _ => {
+	let nav;
+
+	this.timeout(10000);
+
+	afterEach(function () {
+		if (nav) {
+			nav.close();
+		}
+		nav = null;
+	});
+
+	it('open/close events', finish => {
+        const window = Ti.UI.createWindow();
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.navigationwindow.addontest.js;
  Line: 24;
-->
- :no_entry_sign: tests/Resources/ti.ui.navigationwindow.addontest.js line 24 – Expected indentation of 2 tabs but found 8 spaces. (indent)